lib/dynamic_config/dcdo.rb

```diff
@@ -55,6 +55,8 @@ def frontend_config
       'student-email-post-enabled': DCDO.get('student-email-post-enabled', false),
       'progress-v2-metadata-enabled': DCDO.get('progress-v2-metadata-enabled', false),
       'show-updated-lms-content': DCDO.get('show-updated-lms-content', false),
+      # Used to test DCDO setting via cookie. See: 'dashboard/test/ui/features/cookie_dcdo.rb'
+      cookie_dcdo_test: DCDO.get('cookie_dcdo_test', nil),
```

Comment:
Do you even need to put this in here? I'm not seeing exactly now the UI test referenced there needs to put the value in the JS DCDO space. I admit about 50% of this mechanism is still magic to me, haha, so who knows.

Reply:
It's just a straightforward method to verify that the functionality is working and that the DCDO returns the expected value.
https://github.com/code-dot-org/code-dot-org/blob/661fff3a9f6f822fecf7e1407941017da656a379/dashboard/app/views/layouts/application.html.haml#L26-L28

Reply:
Oh! I see! That makes sense. 👍